net/netfilter/nf_tables_api.c

```diff
@@ -9032,11 +9064,21 @@ static void __net_exit nf_tables_pre_exit_net(struct net *net)
 
 static void __net_exit nf_tables_exit_net(struct net *net)
 {
+	struct nftables_pernet *nft_net = nft_pernet(net);
+	unsigned int gc_seq;
+
 	mutex_lock(&net->nft_commit_mutex);
+
+	gc_seq = nft_gc_seq_begin(nft_net);
+
 	if (!list_empty(&net->nft.commit_list) ||
 	    !list_empty(&net->nft_module_list))
 		__nf_tables_abort(net, NFNL_ABORT_NONE);
+
 	__nft_release_tables(net);
+
+	nft_gc_seq_end(nft_net, gc_seq);
+
 	mutex_unlock(&net->nft_commit_mutex);
```

Comment:
This should be a part of an earlier commit:
`netfilter: nf_tables: fix GC transaction races with netns and netlink event exit path`
https://github.com/ctrliq/kernel-src-tree/commit/6a33d8b73dfac0a41f3877894b38082bd0c9a5bc#diff-233a9dea2c513f5d208bba55e0f8ef4e8a29f5c584d30b5815cfa6c2cf5361ecR9742

Reply:
OK, I've moved the wandering code into the correct commit - there is still some white space fuzz but the lines of code match up to the commit they belong in.

Reply:
The cherry pick of "add nft_pernet() helper function" just caused a huge mess.  I think it's all fixed up now.